cosmopolitan/migrations/0006_auto_20170525_1626.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.2 on 2017-05-25 16:26
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cosmopolitan', '0005_polygon'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='city',
+            name='slug',
+            field=models.CharField(db_index=True, max_length=200, null=True),
+        ),
+        migrations.AlterField(
```

Comment:
Added by the autogeneration of the migration. Maybe was uncomitted by previous alteration?